users/utils.py

```diff
@@ -33,7 +37,12 @@ def _reformat_for_username(string):
     Returns:
         str: A version of the string with username-appropriate characters
     """
-    cleaned = re.sub(USERNAME_INVALID_CHAR_PATTERN, "", string)
+    turkish_chars_removed = re.sub(
+        USERNAME_INVALID_TURKISH_CHARS,
+        USERNAME_INVALID_TURKISH_CHARS_REPLACEMENT,
+        string,
+    )
+    cleaned = re.sub(USERNAME_INVALID_CHAR_PATTERN, "", turkish_chars_removed)
```

Comment:
```suggestion
    cleaned_string = re.sub(USERNAME_INVALID_CHAR_PATTERN, "", string)
    cleaned_string = re.sub(
        USERNAME_INVALID_TURKISH_CHARS,
        USERNAME_INVALID_TURKISH_CHARS_REPLACEMENT,
        cleaned_string,
    )
```

This is a nitpick, but I think it's appropriate to remove the other invalid characters first, then perform the turkish "i" replacement since that's much more of an edge case.

Reply:
Sure, will update this

Reply:
Done